src/DebuggerForm.cpp

```diff
@@ -1083,9 +1083,9 @@ void DebuggerForm::searchGoto()
 {
 	GotoDialog gtd(memLayout, &session, this);
 	if (gtd.exec()) {
-		int addr = gtd.address();
-		if (addr >= 0) {
-			disasmView->setCursorAddress(addr, 0, DisasmViewer::MiddleAlways);
+		auto addr = gtd.address();
+		if (addr) {
```

Comment:
detail: limit the scope of the local variable:
`if (auto addr = gtd.address()) {`